front/src/styles/scss/applications/operationalStudies/_pacedTrain.scss

```
@@ -0,0 +1,143 @@
+.paced-train {
+  align-items: center;
+  padding: 12px 0px;
+
+  .paced-train-frequency {
```

To be aligned with the name we use in the .tsx ? (same for the `.frequency` class).
Or since this element doesn't cover the cadence only but also the icons for too fast / not honored, it could be a more generic name like `right-zone` (the name used on the mockup to mention the zone where the action buttons pops when hovering it) or something else
```suggestion
  .paced-train-cadence {
```

---

Done